airbyte-webapp/src/packages/cloud/services/auth/reducer.ts

```diff
@@ -57,4 +58,16 @@ export const authStateReducer = createReducer<AuthServiceState, Actions>(initial
       emailVerified: false,
       loggedOut: true,
     };
+  })
+  .handleAction(actions.updateUserName, (state, action): AuthServiceState => {
+    if (!state.currentUser) {
+      return state;
+    }
```

Comment:
As mentioned in a separate comment, this should never happen if we check that the current user exists first.